src/reporting/IDisputeBond.sol

```diff
@@ -0,0 +1,12 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/Typed.sol';
+import 'ROOT/reporting/IMarket.sol';
+
+
+contract IDisputeBond is Typed {
```

Comment:
Perhaps we should either rename DisputeBondToken to DisputeBond to match this, or rename this to IDisputeBondToken.sol?

Reply:
The long-term plan is to rename `DisputeBondToken` to `DisputeBond` and make it no longer a token and instead an `Ownable`.  However, I wanted to wait until after this PR to address that (trying to avoid growing this PR any more than necessary).